apps/webapp/src/app/utils/balance.ts

```diff
@@ -0,0 +1,20 @@
+export function getLowBalanceStatus({
+  portfolioBalanceWei,
+  minRequiredTotalWei,
+  minTradeThresholdWei
+}: {
+  portfolioBalanceWei: bigint;
+  minRequiredTotalWei: bigint;
+  minTradeThresholdWei: bigint;
+}) {
+  const isBelowMinRebalanceLimit = portfolioBalanceWei < minTradeThresholdWei;
+  const isLowBalance = portfolioBalanceWei < minRequiredTotalWei;
+  const hasNoAssetsAndNoTargets =
+    portfolioBalanceWei === BigInt(0) && minRequiredTotalWei === BigInt(0);
+
+  return {
+    isBelowMinRebalanceLimit,
```

Comment:
naming inconsistency based on variables this variable is evaluating `...limit` > `...threshold`, `...rebalance` > `...trade`
```suggestion
    isBalanceBelowMinTradeThreshold,
```